packages/analytics-adapter/src/data-layer-adapter.ts

```diff
@@ -0,0 +1,44 @@
+import {Awaiter} from "./awaiter";
+
+export abstract class DataLayerAdapter extends Awaiter {
+    evolvReady = false;
+    dataToApplyWhenEvolvReady: {[key:string]: any}[] = [];
```

Comment:
Array of dictionaries, is that correct?
```suggestion
    dataToApplyWhenEvolvReady: Record<string, any>[] = [];
```